src/styles/common-styles.module.scss

```diff
@@ -1,69 +1,66 @@
-@import './theme.module.scss';
+@import './colors.scss';
 
 .intro-button {
```

Comment:
Missing import of `typography.scss` so all variables that come from there aren't affecting anything.
Might need a UX when you do import it as stuff looks a bit differently. (or maybe just don't use the variables and don't change this rules?)